khmer/_khmermodule.cc

```diff
@@ -466,6 +466,20 @@ ReadParser_iter_reads(PyObject * self, PyObject * args )
     return PyObject_SelfIter( self );
 }
 
+static
+PyObject *
+ReadParser_get_num_reads(khmer_ReadParser_Object * me, PyObject * args)
+{
```

Comment:
`readparser.get_num_reads()` is a bit Java-y. You can implement this value as a simple attribute (accessed via `readparser.num_reads` via `tp_members` (https://docs.python.org/2/extending/newtypes.html#generic-attribute-management)